superset/config.py

```diff
@@ -1125,9 +1128,9 @@ class CeleryConfig:  # pylint: disable=too-few-public-methods
 GLOBAL_ASYNC_QUERIES_TRANSPORT = "polling"
 GLOBAL_ASYNC_QUERIES_POLLING_DELAY = 500
 
-# It's possible to add a dataset health check logic which is specific to your system.
-# It will get executed each time when user open a chart's explore view.
-DATASET_HEALTH_CHECK = None
+# A SQL dataset health check. Note if enabled the callable should be cached to aid with
+# performance.
```

Comment:
1. If the healthy status is in `cache` but not persist in database, when restart server will we lost all the status?
2. right now we can run some queries to see healthy status. if use `cache` solution we will not be easy to know this number?

Reply:
@graceguo-supercat:

1. The Redis (or similar) cache rarely is restarted.
2. It's definitely less clear though possible via a Redis (or similar) query.